_posts/2023-03-17-introduction-to-discrete-event-simulations-in-healthcare/introduction-to-discrete-event-simulations-in-healthcare.Rmd

```diff
@@ -0,0 +1,112 @@
+---
+title: "Introduction to Discrete Event Simulations in Healthcare"
+description: |
+  An introduction to Discrete Events Simulations (DES) in healthcare with an overview of the use of DES for modelling outpatient services appointments at Providence Healthcare.
+  
+author:
+  - name: Kevin Wang, Neil Mistry
+    url: https://www.linkedin.com/in/mingkun-wang/, https://www.linkedin.com/in/neilmistry/
+date: 2022-09-23
+output:
+  distill::distill_article:
+    self_contained: false
+draft: true
+category: post-miscellaneous
+---
+
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = FALSE)
+```
+
+# Discrete Events Simulation at Providence Outpatient Care & Services Covid-19 Recovery scheduling
+
+
+Discrete Events Simulations (DES) are a method of modeling the operation of real world systems digitally.  It does this by modeling a series of events in time.  Each event is ordered by time of occurance, and the system is advanced to each event in the sequence, meanwhile updating the state of the system.  This is done so that we can test hypothetical changes to the system and their impacts on metrics we are interested in, uncover bottlenecks or process challenges and make overall assessments of the system.
+
+This type of simulation is particularly useful at modeling a system with many uncertain or probabalistic steps (such as you don't know when your next customer or patient will arrive, and you don't know how long and what resources are required to help that customer or patient.)  In sytems where there a complex number of steps, many different resources, inputs and outputs, DES may be one of the only possible ways to model.
+The underlying statsical theroes that support DES come from queueing theory.  A classic example of queueing theory is the call center.  In a call center you have incoming calls.  You may know roughly how many calls you can expect on a day, or any given hour, and hence you know the frequency of calls.  
+
+![Poisson Call Center Queue](./queueing_example_possion_call_center.jpg)
+
+However you do not know exactly when your next call will arrive (remember this is probabiltic or uncertain).  This is known as a poisson arrival process, and is modeled as:
+
+![Poisson Call Center Queue](./PoissonFormula.gif)
+
+Lambda is the only paramter that Poisson depends on.  This is your arrival rate (on average.)  
```

Comment:
```suggestion
Lambda is the only parameter that Poisson depends on.  This is your arrival rate (on average.)  
```

Reply:
Not sure if this will work in markdown, but maybe you can try `\lambda` instead of `Lambda`, so that it can render the actual symbol.

Reply:
For RMD you may need the `tinytex` package and to do something like $\Lambda$

Reply:
woops... that actually rendered when I typed it. you need something like $ \Lambda $, no spaces between $ and the next character